style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  min-height: 100vh;
+  background: #666;
+  justify-content: center;
+  align-items: center;
+  font-family: "Lato", sans-serif;
+  font-weight: 400;
+ }
+
+.calculator {
+  position: relative;
+  height: 525px;
+  width: 320px;
+  border-radius: 10px;
+  background: #222;
+  overflow: hidden;
+  box-shadow: 4px 2px 20px rgba(0, 0, 0, 0.5);
+}
+
+.screen {
+  width: 100%;
+  height: 130px;
+  border: none;
+  border-radius: 10px;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+  font-size: 45px;
+  padding: 15px;
+  color: #fff;
+  background: #000;
+  text-align: right;
+  pointer-events: none;
+  font-family: "Doto", sans-serif;
+  font-weight: 700;
+}
+
+.error-message {
+  position: absolute;
```

Comment:
Never use `position: absolute` if it's not absolutely necessary. It makes the layout a lot more complicated and unpredictable.